content/en/dashboards/_index.md

```diff
@@ -204,6 +204,24 @@ An organization's most popular dashboard displays five popularity bars. All othe
 
 **Note**: Traffic to public dashboard URLs is ignored for popularity.
 
+## Viewing dashboards on mobile devices
+
+The [Datadog Mobile App][1], available on the [Apple App Store][2] and [Google Play Store][3], enables you to view your dashboards in a mobile-friendly format.
+
+On the Dashboards page, you can view and search all of your dashboards, and filter them using the same template variables you have set up in the Datadog web app. Quickly filter your dashboards using template variable saved views. For more information about template variable saved views, see [Dashboard Saved Views][14]. Click on an individual dashboard to view it.
+
+**Note**: To set up or edit a dashboard, you need to log in to the Datadog web app. For more information, see [Dashboards][4].
```

Comment:
```suggestion
**Note**: To set up or edit a dashboard, you must log in to Datadog. For more information, see [Dashboards][4].
```

Reply:
I'd like to keep the language around the web app specifically, since we need to be clear that dashboards can't be created from mobile. Is there a better way to refer to the desktop web app specifically?

Reply:
@conward2121 instead of "the Datadog web app", I'd suggest "the Datadog browser UI"